packages/quantic/force-app/main/default/lwc/quanticRecommendationList/e2e/fixture.ts

```diff
@@ -0,0 +1,35 @@
+import {RecommendationListObject} from './pageObject';
+import {SearchObject} from '../../../../../../playwright/page-object/searchObject';
+import {quanticBase} from '../../../../../../playwright/fixtures/baseFixture';
+import {searchRequestRegex} from '../../../../../../playwright/utils/requests';
+
+const pageUrl = 's/quantic-recommendation-list';
+
+interface RecommendationListOptions {
+  numberOfRecommendations: number;
+  recommendationsPerRow: number;
+  variant: string;
+}
+
+type QuanticRecommendationListE2EFixtures = {
+  recommendationList: RecommendationListObject;
+  search: SearchObject;
+  options: Partial<RecommendationListOptions>;
+};
+
+export const testSearch =
```

Comment:
you can just call this `test`, first because this is not the search use case, and second because there is only one fixture in this file.